### PR TITLE
Setup CCD Diff

### PR DESCRIPTION
**Pull request by linusnorton**
@DelythJustice I've set up CCD diff. Once merged into master PRs will receive a report on the PR like this one: https://github.com/hmcts/rse-ccd-diff/pull/4#issuecomment-1163538298

Everyone's JSON file structure is a little bit different but I believe it will work with yours. Let me know if there are any issues or you notice that it's not picking up some changes.

### CODE REVIEW

**Comment by DelythJustice** [comment]
Thanks very much for this. Might need to wait until Monday to merge, business usually block fridays.

**Review by sanjayparekh-hmcts — APPROVED** [review]

**Review by DelythJustice — APPROVED** [review]
fab

**Pull request merged**
